src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdmin.cshtml

```diff
@@ -15,8 +19,21 @@
     {
         <script asp-src="~/OrchardCore.Lists/Scripts/ci-list-ordering.min.js" debug-src="~/OrchardCore.Lists/Scripts/ci-list-ordering.js" asp-name="ci-list-ordering" at="Foot" depends-on="sortable, jQuery"></script>
     }
-}
 
+    var authorizedIndexes =
+        await Task.WhenAll(
+            Model.ContainedContentTypeDefinitions.
+                Select(async contentTypeDefinition =>
+                    await AuthorizationService.AuthorizeAsync(
+                        User,
+                        CommonPermissions.EditContent,
+                        ContentManager.NewAsync(contentTypeDefinition.Name))));
```

Comment:
Instead of creating an enumerable of content items here, you can change this to return the content definitions only where the corresponding item is authorized.